keras_cv/layers/preprocessing/mix_up.py

```diff
@@ -110,27 +110,40 @@ def _update_labels(self, labels, lambda_sample, permutation_order):
         return labels
 
     def _update_bounding_boxes(self, bounding_boxes, permutation_order):
-        boxes_for_mixup = tf.gather(bounding_boxes, permutation_order)
-        bounding_boxes = tf.concat([bounding_boxes, boxes_for_mixup], axis=1)
-        return bounding_boxes
+        boxes, classes = bounding_boxes["boxes"], bounding_boxes["classes"]
+        boxes_for_mixup = tf.gather(boxes, permutation_order)
+        classes_for_mixup = tf.gather(classes, permutation_order)
+        boxes = tf.concat([boxes, boxes_for_mixup], axis=1)
+        classes = tf.concat([classes, classes_for_mixup], axis=1)
+        return {"boxes": boxes, "classes": classes}
 
     def _validate_inputs(self, inputs):
         images = inputs.get("images", None)
         labels = inputs.get("labels", None)
         bounding_boxes = inputs.get("bounding_boxes", None)
+
         if images is None or (labels is None and bounding_boxes is None):
             raise ValueError(
                 "MixUp expects inputs in a dictionary with format "
                 '{"images": images, "labels": labels}. or'
                 '{"images": images, "bounding_boxes": bounding_boxes}'
-                f"Got: inputs = {inputs}"
+                f"Got: inputs = {inputs}."
             )
+
         if labels is not None and not labels.dtype.is_floating:
             raise ValueError(
                 f"MixUp received labels with type {labels.dtype}. "
                 "Labels must be of type float."
             )
 
+        if bounding_boxes is not None:
```

Comment:
Should this check be around for all layers?

Reply:
I guess `bounding_box.validate_format()` handles this now, so we don't need this anymore - updated to use the utility.